protocol/app/msgs/all_msgs.go

```diff
@@ -342,6 +342,65 @@ var (
 		"/ibc.applications.interchain_accounts.controller.v1.MsgUpdateParamsResponse":              {},
 		"/ibc.applications.interchain_accounts.host.v1.MsgUpdateParams":                            {},
 		"/ibc.applications.interchain_accounts.host.v1.MsgUpdateParamsResponse":                    {},
+
+		// wasm messages
+		"/cosmwasm.wasm.v1.AcceptedMessageKeysFilter":                  {},
```

Comment:
Can you add a TODO ticket to audit if all messages are needed? We want to support the bare minimal necessary

Reply:
Will do

Reply:
Can you comment inline `TODO(linear-ticket)`